concepts/05 Widgets/LoadIndicator/05 Show and Hide Using the API.md

```diff
@@ -12,14 +12,14 @@ To specify whether the **LoadIndicator** is shown, change the [visible](/api-ref
 
     <!--JavaScript-->
     $(function() {
-        var loadIndicator =  $("#loadIndicatorContainer").dxLoadIndicator({
+        const loadIndicator =  $("#loadIndicatorContainer").dxLoadIndicator({
             visible: false      
         }).dxLoadIndicator("instance");
         
         $("#buttonContainer").dxButton({
             text: "Toggle the LoadIndicator", 
             onClick: function () {
-                var isLoadIndicatorVisible = loadIndicator.option("visible");
+                const isLoadIndicatorVisible = loadIndicator.option("visible");
```

Comment:
@ksercs Isn't this option designed to be changed at runtime? Seems like `let` fits better than `const`.

Reply:
it's local constant existing only inside of `onClick` scope, so there is no reinit or assignment